src/pybind/mgr/cephadm/services/ingress.py

```diff
@@ -112,7 +113,9 @@ def get_haproxy_dependencies(mgr: "CephadmOrchestrator", spec: Optional[ServiceS
             if ssl_cert_key:
                 assert isinstance(ssl_cert_key, str)
                 deps.append(f'ssl-cert-key:{str(utils.md5_hash(ssl_cert_key))}')
-
+        if ingress_spec.backend_service.split('.')[0] == 'nfs':
```

Comment:
NIT: this almost certainly would work, but I'd prefer we're consistent about how we decide the backend service type. Elsewhere we effectively do
```
backend_spec = self.mgr.spec_store[spec.backend_service].spec
if backend_spec.service_type == 'nfs':
```